tests/common/dualtor/dual_tor_utils.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import pytest
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope='session')
+def tor_mux_intf(duthosts):
+    '''
+    Returns the server-facing interface on the ToR to be used for testing 
+    '''
+    # The same ports on both ToRs should be connected to the same PTF port
+    dut = duthosts[0]
+    return sorted(dut.get_vlan_intfs(), key=lambda intf: int(intf.replace('Ethernet', '')))[0]
+
+
+@pytest.fixture(scope='session')
```

Comment:
Fixture `duthosts` has scope 'module'. If I remember correctly, pytest would not allow constructing a bigger scope fixture based on smaller scope fixtures.

Reply:
I had also thought this, but I'm able to use this fixture without any issue.